docker-compose.yml

```diff
@@ -17,8 +17,8 @@ services:
     restart: unless-stopped
     environment:
       - ROOT_URL=http://localhost
-      - MONGO_URL=mongodb://mongo:27017/rocketchat?replicaSet=rs01
-      - MONGO_OPLOG_URL=mongodb://mongo:27017/local?replicaSet=rs01
+      - MONGO_URL=mongodb://mongo:27017/rocketchat?replSet=rs01
```

Comment:
this isn't correct

https://docs.mongodb.com/manual/reference/connection-string/#connection-string-options